website/content/docs/configuration/client.mdx

```diff
@@ -352,9 +352,97 @@ see the [drivers documentation](/docs/drivers).
   the host as root (unless Nomad is configured to run as a non-root user).
 
 - `disable_file_sandbox` `(bool: false)` - Allows templates access to arbitrary
-  files on the client host via the `file` function. By default templates can
+  files on the client host via the `file` function. By default, templates can
   access files only within the [task working directory].
 
+- `max_stale` `(string: "")` - # This is the maximum interval to allow "stale"
+  data. By default, only the Consul leader will respond to queries. Requests to
+  a follower will forward to the leader. In large clusters with many requests,
+  this is not as scalable. This option allows any follower to respond to a query,
+  so long as the last-replicated data is within this bound. Higher values result
+  in less cluster load, but are more likely to have outdated data.
+
+- `wait` `(Code: nil)` - Defines the minimum and maximum amount of time to wait
+  for the Consul cluster to reach a consistent state before rendering a template.
+  This is useful to enable in systems where Consul is experiencing a lot of flapping,
```

Comment:
We use the phrase "a lot of flapping" in this PR and although it's nice and pithy, it also feels like slang. Maybe replace with "where network connectivity to Consul is degraded"?

Reply:
Done